README.md

```diff
@@ -263,20 +274,25 @@ Returns the currently logged-in user's Facebook user ID.
 ---------------------------------------
 
 <a name="sendMessage" />
-### api.sendMessage(message, thread_id, [callback])
+### api.sendMessage(message, threadID, [callback])
 
 Sends the given message to the thread_id.
 
 __Arguments__
 
-* `message` - A string (for backward compatibility) or a message object as described below.
-* `thread_id` - A string, number, or array representing a thread. It happens to be someone's userId in the case of a one to one conversation or an array of userIds when starting a new group chat.
-* `callback(err, threadID)` - A callback called when sending the message is done (either with an error or with an confirmation object). `threadID` contains only the thread_id where the message was sent.
+* `message`: A string (for backward compatibility) or a message object as described below.
+* `threadID`: A string, number, or array representing a thread. It happens to be someone's userId in the case of a one to one conversation or an array of userIds when starting a new group chat.
+* `callback(err, messageInfo)`: A callback called when sending the message is done (either with an error or with an confirmation object). `messageInfo` contains the `threadID` where the message was sent and a `messageID`.
+
+__Message Object__: 
 
-*Message Object*: Various types of message can be sent:
-* Regular - Set a field `body` to the desired message.
-* Sticker - Set a field `sticker` to the desired sticker ID.
-* File/Image - Set field `attachment` to a readable stream or an array of readable streams.
+Various types of message can be sent:
+* *Regular:* set field `body` to the desired message as a string.
+* *Sticker:* set a field `sticker` to the desired sticker ID.
+* *File or image:* Set field `attachment` to a readable stream or an array of readable streams.
+* *URL:* set a field `url` to the desired URL.
+
+Note that a message can only be a regular message (which can be empty) and optionally one of the following: a sticker, an attachment or a url.
 
 __Tip__: to find your own ID, go to your own profile on Facebook and replace 'www' by 'graph' in the URL.
```

Comment:
This actually doesn't work anymore :/. A replacement could be: go to your profile, click on your profile picture, look at the url that will look like `https://www.facebook.com/photo.php?fbid=10206056682155550&set=a.1652922442382.91979.|XXXXXXXXXX|&type=1&theater` where the `|XXXXXXXXXX|` are your `ID`.